### ISSUE TITLE
Deprecate BintrayPublisher and default to SonatypePublisher

### ISSUE THREAD

**Issue by wsargent** [scalacenter/sbt-release-early#39]
jFrog is sunsetting Bintray https://jfrog.com/blog/into-the-sunset-bintray-jcenter-gocenter-and-chartcenter/

So it's probably time to shift people to using Sonatype.

**Comment by jvican** [comment]
I agree we probably want to remove Maven Central support for Bintray. However, maybe some people still want to publish to Bintray's internal Artifactory? I have no strong opinion but given that we support that we probably don't want to remove it.

Any contribution to clean up the bintray publishing is welcome, I don't have any free cycles available right now.

**Comment by wsargent** [comment]
> I don't have any free cycles available right now.

same tbh
